web/src/links.jsx

```diff
@@ -5,41 +5,51 @@ import {
   Edit,
   Create,
   SimpleForm,
+  FunctionField,
   TextField,
   EditButton,
   TextInput,
   required,
 } from "react-admin";
 
+const PREFIX = "https://links.zenika.com/link"
```

Comment:
Let's use an env var here, because it should be different in local dev (`http://localhost:3000/link`) and prod (`https://links.zenika.com/link`).